src/styles/main.scss

```diff
@@ -0,0 +1,78 @@
+.page {
+  margin: 0;
+
+  &__watch {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    width: 600px;
+    height: 600px;
+  }
+}
+
+.watch {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &__seconds {
+    position: absolute;
+    left: 30px;
+    width: 270px;
+    height: 10px;
+    background-color: lightseagreen;
+    border-radius: 15px;
+
+    animation-name: seconds;
+    animation-duration: 60s;
+    animation-iteration-count: infinite;
+    animation-timing-function: linear;
+
+    @keyframes seconds {
+      from {
+        transform: rotate(86deg);
+        transform-origin: right;
+      }
+
+      to {
+        transform: rotate(446deg);
+        transform-origin: right;
+      }
+    }
+  }
+
+  &__minutes {
+    position: absolute;
+    right: 100px;
+    width: 200px;
+    height: 15px;
+    background-color: rgb(255, 127, 110);
+    border-radius: 15px;
+
+    animation-name: minutes;
+    animation-duration: 3600s;
+    animation-iteration-count: infinite;
+    animation-timing-function: linear;
+
+    @keyframes minutes {
+      from {
+        transform: rotate(270deg);
+        transform-origin: left;
+      }
+
+      to {
+        transform: rotate(630deg);
+        transform-origin: left;
+      }
+    }
```

Comment:
it is customary to write animation at the end of the file and not nest it in any selector